### DISCUSSION TITLE
How to support a regular string contains interpolation syntax?

### DISCUSSION THREAD

**Discussion by Nic-Ma** [omry/omegaconf#886] [Q&A]
**Is your feature request related to a problem? Please describe.**
Thanks for the great project, very interesting and creative.
I see you can support interpolation with `${XXX}` syntax, is there a way to avoid errors if a regular string occasionally contains the syntax? For example:
`"rand_password": "Dfw%$#${*(d##@E}@*^fs"`

Thanks in advance.

**Comment by Jasha10** [comment]
Hi @Nic-Ma,
You will need to use a backslash escape `'\'` for the `'${'` characters:
```yaml
"rand_password": "Dfw%$#\${*(d##@E}@*^fs"
```
See the documentation on [escaped characters](https://omegaconf.readthedocs.io/en/latest/grammar.html#escaped-characters) in interpolation strings.
